SitecorePackageDeployer.Web/Hhogdev.SitecorePackageDeployer.Web.csproj

```diff
@@ -13,7 +13,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Hhogdev.SitecorePackageDeployer.Web</RootNamespace>
     <AssemblyName>Hhogdev.SitecorePackageDeployer.Web</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
Will this cause conflicts? Looking at vanilla installs of 8.2uX- looks like 4.5.2 is targeted. Now, since this is rather independent, maybe it will be fine.